worker/src/main/scala/org/apache/celeborn/service/deploy/worker/WorkerSource.scala

```diff
@@ -72,6 +72,8 @@ class WorkerSource(conf: CelebornConf) extends AbstractSource(conf, MetricsSyste
   addTimer(REPLICA_REGION_START_TIME)
   addTimer(PRIMARY_REGION_FINISH_TIME)
   addTimer(REPLICA_REGION_FINISH_TIME)
+  addTimer(PRIMARY_SEGMENT_START_TIME)
```

Comment:
The introduced metrics could create another pull request, otherwise please add these metrics in document and `celeborn-dashboard.json`.